### ISSUE TITLE
Transactions syncing issues

### ISSUE THREAD

**Issue by gabmontes** [autonomoussoftware/metronome-wallet-desktop#495]
The wallet queries the ETC indexer and retrieves a list of transactions but none are displayed in the transactions list.

In addition, some ETH transactions like ports are not properly parsed and just shown as generic transfers or contract interactions.

**Comment by gndelia** [comment]
This seems to be a problem with the nodes we're using to sync. It is failing to retrieve some of the Txs. We'll look at the details of why the nodes are not retrieving properly those Txs.

**Comment by gabmontes** [comment]
That's correct. The ETC nodes were not returning some transactions so the sync failed. Not a problem with the wallet.

For ETH port transactions, it looks like there is a race condition that is preventing the transactions to be properly parsed. If one of such transactions is refreshed individually (open transaction details, click "Refresh"), all the data is properly parsed.
